CMakeLists.txt

```diff
@@ -249,24 +284,30 @@ else(MSVC)
     set(CMAKE_RDYNAMIC_FLAG "-rdynamic")
   endif(CMAKE_COMPILER_IS_GNUCC)
 
-  set(CMAKE_CXX_FLAGS                 "-std=c++11 -pthread ${CMAKE_GCC_FLAGS} -fPIC ${DISABLE_GLOBALLY} -march=${BUILD_ARCH} ${INTRINSICS}")
-  set(CMAKE_CXX_FLAGS_RELEASE         "-O3 -m64 -funroll-loops -g ${CMAKE_RDYNAMIC_FLAG}")
+  set(CMAKE_CXX_FLAGS                 "-std=c++11 -pthread ${CMAKE_GCC_FLAGS} -fPIC ${DISABLE_GLOBALLY} ${INTRINSICS}")
```

Comment:
I wonder why we were using `-march` and if it's okay to drop it in all builds (probably yes).

Reply:
probably in there for additional optimizations for people building & running marian locally. if you confirm I can take it out then I can simplify the logic a bit

Reply:
@snukky @hieuhoang would you prefer that I leave the -march in or take it out?

Reply:
I am going to leave it as is right now to not break any backwardscompat. if we want, we can take it out later